cmd/syft/internal/options/catalog.go

```diff
@@ -259,6 +259,9 @@ func (cfg *Catalog) AddFlags(flags clio.FlagSet) {
 
 	flags.StringVarP(&cfg.Source.BasePath, "base-path", "",
 		"base directory for scanning, no links will be followed above this directory, and all paths will be reported relative to this directory")
+
+	flags.StringVarP(&cfg.Source.Supplier, "source-supplier", "",
+		"a supplier may also be known as a manufacturer, organization, vendor, developer, integrator, maintainer, or provider of the software")
```

Comment:
Sorry to nitpick this one, but what I meant about organization here was that this field actually should NOT be developer, maintainer, etc., it is explicitly only the organizational entity which is identified as the supplier. This is because it is never output as an SPDX _person_, and the [CDX supplier explicitly says it is an organization](https://cyclonedx.org/docs/1.6/json/#components_items_supplier), so we do not support people here, only organizations.

Reply:
How about more-or-less the CDX text?

> the organization that supplied the component, which often may be the manufacturer, distributor, or repackager.